Keithley2400Sup/Keithley2400.proto

```diff
@@ -33,14 +33,14 @@ getID {
 # /// Read the voltage from the data string. Format is %f (V), %f (I), %f (R), %f (Timestamp), %f (Status)
 get_V {
    ExtraInput = Ignore;
-   out ":READ?";
+   out ":MEAS:VOLT?";
    in "%f,%*f,%*f";
```

Comment:
The rest of the protocol has been changed to use `%g` format converters which handle exponentials. Does this command also want to be changed to using `%g`?